code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
+
+		if("spin")
+			spin(20, 1)
+			message = "<b>[src]</b> spins around dizzily!"
+
+		if("highfive")
+			if(restrained())
+				return
+			if(highfiving)
+				to_chat(src, "You give up on the high-five.")
+				highfiving = FALSE
+				return
+			highfiving = TRUE
+			for(var/mob/living/carbon/C in orange(1))
+				if(C.highfiving)
+					if(C.mind.special_role == "Wizard" && mind.special_role == "Wizard")
```

Comment:
This isn't meant to be something common. This is a highly destructive high-five of immense power, befitting Ragin' Mages levels of rarity.

Otherwise, you'd see this become a common practice for wizard rounds: buy apprentice scroll, high-five the station into non-existence. That's not exactly a fun meta to run with, just like all-captain crews during war ops. Sure, it might work, but it is kinda defeating the intent of the challenge.